datalad_deprecated/resources/website/assets/css/main.css

```diff
@@ -1,6 +1,10 @@
 /*
  * DataLad Static Index styles
  */
+table a { color: inherit; text-decoration: inherit; }
+#folder-link { float: left; margin: 0.3em 1.3em .3em .3em; }
+#folder-button { width: 80px; height: 30px; }
+.visit-folder { box-sizing: border-box; background-color: lightgray; border-radius: 5px; padding: 5px; }
```

Comment:
Button and this styling is no longer needed, right?

Reply:
Could you please then update copy on http://datasets-dev.datalad.org/ to see how it all looks

Reply:
I added some more styling just to differentiate things, I can remove if needed, but yes the button styling should be removed. Ill update in a second.

Reply:
updated the dev site, will push some other commits as well

Reply:
I can make the visit folder a little more fancy than this, though let me know if this base idea looks fine.

Reply:
yeah, not sure about that grey background, but also I guess we can revamp styling later.  Let's proceed meanwhile -- will merge now (and later manually update on datasets.datalad.org). Thank you!